pom.xml

```diff
@@ -47,10 +47,13 @@
 
   <properties>
     <!-- alphabetized properties for versions -->
+    <version.guava>15.0</version.guava>
```

Comment:
I wonder if we can remove this dependency. I try to not use Guava for standalone open source libraries (for example I avoided using it for avro-sasl-netty library, which might be open sourced). Guava has a lot of backwards incompatibilities and when others try to use this they may hit Guava compatibility issues. So I prefer to avoid it for libraries that will be shared publicly.